bundles/AdminBundle/Controller/Searchadmin/SearchController.php

```diff
@@ -526,8 +526,8 @@ protected function shortenPath($path)
             array_splice($parts, $i - 1, 1, '...');
         }
 
-        if (strlen($shortPath) > 50) {
-            $shortPath = substr($shortPath, 0, 47) . '...';
+        if (mb_strlen($shortPath) > 50) {
+            $shortPath = mb_strstr($shortPath, 0, 47) . '...';
```

Comment:
```suggestion
            $shortPath = mb_strstr($shortPath, 0, 49) . '…';
```

Reply:
An actual ellipsis 😄

Reply:
Changed it also for the path a few lines above 😊